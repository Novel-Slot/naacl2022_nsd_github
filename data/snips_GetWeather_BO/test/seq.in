will it snow in mt on june 13  2038
what s the weather here on 2/7/2021
will the wind die down at my current location by supper time
will it snow in amy
what will the weather be at nine am in hi
what will the weather be in lago vista on october fourteenth  2022
weather next year in canada
what is the weather of east portal ks
what is the weather going to be like in klondike gold rush national historical park on february the 28th  2034
what will the weather be in berville ak on feb  6  2017
is it warm in botna
what is the weather forecast for my current place
will it be freezing in the current position
will there be snowfall at six pm in leisure knoll california
tell me when it will be chilly in chicken  united kingdom
is it windy in telogia
check the forecast for the current spot in the future oct  19  2037
is fog forecast close-by to pakistan
is there snow in the district of columbia
what is the forecast in heber
what is the forecast for in 1 second at monte sereno for freezing temps
show me the weather forecast for the city of spencer
how is the weather in getzville  minnesota
i need a weather forecast for são tomé and príncipe on december 8th  2026
tell me the weather forecast for april 15  2019 here
at meal time while i m here will it be hot
will it snowstorm in long lake national wildlife refuge
is it freezing in colorado
will it be stormy in ma
what will the weather be in la at 9 o clock
will it be hot on orthodox good friday in michigan and close-by
what is the weather like in hurstville
what will the weather be in waverly city brazil on purple heart day
what is the weather forecast in delaware
what s the forecast for belize around meal time
what s the weather in low moor
what is the weather like in north salt lake and afghanistan
what s the weather like in schenectady ma
how is the weather right now at my current place
tell me the weather forecast close by brown county state park for meal time
what s it like in bahrain right now
on jan  the twentieth what will it feel like in ct or the area not far from it
i need a table in uruguay in 213 days when it s chillier
can you tell me the weather forecast for samoa
will there be a blizzard in white house curacao
what will the weather be in grand coteau ut at six pm
will it be freezing here in 9 seconds
forecast for serbia
in hawaii will it be warmer at 3 am
how will the weather be different 5 years from now in waconia
what is the niceville forecast in fm
is there rain now in maine
what is the forecast for montana at dinner
weather for this winter here
will there be a cloud here at 06:50:20
is hail in the weather forecast for monterey bay national marine sanctuary
what will the weather be like this tuesday in the area neighboring rendezvous mountain educational state forest
is it forecast to be warm in doi inthanon national park
will there be snowfall in kitlope heritage conservancy
is it going to be chilly in western sahara in 13 hours
what is the forecast for niger
is it going to snow next year in wv
what s the forecast for my current place at five pm
in 17 minutes  will it be foggy in songimvelo game reserve
how hot will it be in wisconsin on august fourth
will it be chilly in oakdale ok
what s the weather forecast for croatia on jul  25th
whats the weather in ga
will there be cloud coverage in verdery  myanmar
will it snow in granbury
what is the forecast for foggy conditions here in twenty one minutes
weather for ma in the morning
is there a snowstorm in russia
will it be nice on aug  the nineteenth in beda bulgaria
weather for the night time in new mexico
will it be a snowy day in dalcour
what is the weather forecast for close-by burkina
what is the freezing forecast for british virgin islands
will there be a snowstorm in taberville
what is the weather forecast for cistern
is it rainy season in manitou springs
what will the weather be like on january 2nd  2025 in ga
will it be chillier at 06:05:48 in wagener réunion
what is the weather in south bradenton
what is the nh forecast for mexican hat
will it be chillier at my current location in one minute
what is the weather forecast for close-by gu 3 years from now
will it be freezing on 4/20/2038 in american beach nc
i need the wather for next week in the philippines
is it freezing on jun  the 21st in apshawa south africa
will it be temperate in the same area in vi
is it going to get chillier near hocking state forest
is it going to be chillier at 10 pm in texas
what s the weather in timbo
tell me the weather forecast for sugarloaf provincial park ten weeks from now
what is the humidity like in faraway on ak
what will the humidity be in varnado georgia at one am
will it get overcast in la dolores
will it be hotter in wyomissing hills
show weather while sunset in the same area in south carolina
what will the weather be in stelvio national park 1 hour and 1 minute from now
is it going to storm in black rock alaska
weather for beauregard il
will there be alot of wind on march 13th in lost creek bahrain
