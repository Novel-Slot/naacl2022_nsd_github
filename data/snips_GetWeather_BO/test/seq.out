O O B-condition_description O B-state O B-timeRange I-timeRange I-timeRange
O O O O B-current_location O B-timeRange
O O B-condition_description O O O O B-current_location I-current_location O B-timeRange O
O O B-condition_description O B-city
O O O O O O B-timeRange I-timeRange O B-state
O O O O O O B-city I-city O B-timeRange I-timeRange I-timeRange
O B-timeRange I-timeRange O B-country
O O O O O B-city I-city B-state
O O O O O O O O O B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi O B-timeRange I-timeRange I-timeRange I-timeRange
O O O O O O B-city B-state O B-timeRange I-timeRange I-timeRange
O O B-condition_temperature O B-city
O O O O O O O B-current_location I-current_location
O O O B-condition_temperature O O B-current_location I-current_location
O O O B-condition_description O B-timeRange I-timeRange O B-city I-city B-state
O O O O O O B-condition_temperature O B-city B-country I-country
O O B-condition_description O B-city
O O O O O B-current_location I-current_location O O O B-timeRange I-timeRange I-timeRange
O B-condition_description O B-spatial_relation O B-country
O O B-condition_description O O B-state I-state I-state
O O O O O B-city
O O O O O B-timeRange I-timeRange I-timeRange O B-city I-city O B-condition_temperature O
O O O O O O O O O B-city
O O O O O B-city B-state
O O O O O O B-country I-country I-country I-country O B-timeRange I-timeRange I-timeRange
O O O O O O B-timeRange I-timeRange I-timeRange B-current_location
O B-timeRange O O O O B-current_location O O O B-condition_temperature
O O B-condition_description O B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi
O O B-condition_temperature O B-state
O O O B-condition_description O B-state
O O O O O O B-state O B-timeRange I-timeRange I-timeRange
O O O B-condition_temperature O B-timeRange I-timeRange I-timeRange O B-state O B-spatial_relation
O O O O O O B-city
O O O O O O B-city I-city B-country O B-timeRange I-timeRange I-timeRange
O O O O O O B-state
O O O O O B-country O B-timeRange O
O O O O O B-city I-city
O O O O O O B-city I-city I-city O B-country
O O O O O O B-city B-state
O O O O O B-timeRange O O B-current_location I-current_location
O O O O O B-spatial_relation I-spatial_relation B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi O B-timeRange O
O O O O O B-country O B-timeRange
O B-timeRange I-timeRange I-timeRange O O O O O O B-state O O O B-spatial_relation I-spatial_relation O O
O O O O O B-country B-timeRange I-timeRange I-timeRange O O O B-condition_temperature
O O O O O O O O B-country
O O O O B-condition_description O B-city I-city B-country
O O O O O O B-city I-city B-state O B-timeRange I-timeRange
O O O B-condition_temperature B-current_location B-timeRange I-timeRange I-timeRange
O O B-country
O B-state O O O B-condition_temperature O B-timeRange I-timeRange
O O O O O O B-timeRange I-timeRange I-timeRange I-timeRange O B-city
O O O B-city O O B-state
O O B-condition_description B-timeRange O B-state
O O O O O B-state O B-timeRange
O O B-timeRange I-timeRange B-current_location
O O O O B-condition_description B-current_location O B-timeRange
O B-condition_description O O O O O B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi
O O O O O O B-timeRange I-timeRange O O O B-spatial_relation B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi
O O O O O B-condition_temperature O B-geographic_poi I-geographic_poi I-geographic_poi I-geographic_poi
O O O B-condition_description O B-geographic_poi I-geographic_poi I-geographic_poi
O O O O O B-condition_temperature O B-country I-country O B-timeRange I-timeRange
O O O O O B-country
O O O O B-condition_description B-timeRange I-timeRange O B-state
O O O O O O B-current_location I-current_location O B-timeRange I-timeRange
B-timeRange I-timeRange I-timeRange O O O B-condition_description O B-geographic_poi I-geographic_poi I-geographic_poi
O B-condition_temperature O O O O B-state O B-timeRange I-timeRange
O O O B-condition_temperature O B-city B-state
O O O O O O B-country O B-timeRange I-timeRange
O O O O B-state
O O O B-condition_description O O B-city B-country
O O B-condition_description O B-city
O O O O O B-condition_description O B-current_location B-timeRange I-timeRange I-timeRange I-timeRange
O O B-state O O B-timeRange
O O O B-condition_description O B-country
O O O O O B-timeRange I-timeRange I-timeRange O B-city B-country
O O O B-timeRange O O B-state I-state
O O O O B-condition_description B-timeRange O B-city
O O O O O O B-spatial_relation B-country
O O O B-condition_temperature O O B-country I-country I-country
O O O O B-condition_description O B-city
O O O O O O B-city
O O B-condition_description O O B-city I-city
O O O O O O O B-timeRange I-timeRange I-timeRange O B-state
O O O B-condition_temperature O B-timeRange O B-city B-country
O O O O O B-city I-city
O O O B-state O O B-city I-city
O O O B-condition_temperature O O B-current_location I-current_location B-timeRange I-timeRange I-timeRange
O O O O O O B-spatial_relation B-state B-timeRange I-timeRange I-timeRange I-timeRange
O O O B-condition_temperature O B-timeRange O B-city I-city B-state
O O O O O B-timeRange I-timeRange O O B-country
O O B-condition_temperature O B-timeRange I-timeRange I-timeRange O B-city B-country I-country
O O O B-condition_temperature B-spatial_relation I-spatial_relation I-spatial_relation I-spatial_relation O B-state
O O O O O B-condition_temperature B-spatial_relation B-geographic_poi I-geographic_poi I-geographic_poi
O O O O O B-condition_temperature O B-timeRange I-timeRange O B-state
O O O O O B-city
O O O O O O B-geographic_poi I-geographic_poi I-geographic_poi B-timeRange I-timeRange I-timeRange I-timeRange
O O O B-condition_description O O B-spatial_relation O B-state
O O O B-condition_description O O B-city B-state O B-timeRange I-timeRange
O O O B-condition_description O B-city I-city
O O O B-condition_temperature O B-city I-city
O O O B-timeRange B-spatial_relation I-spatial_relation I-spatial_relation I-spatial_relation O B-state I-state
O O O O O O B-geographic_poi I-geographic_poi I-geographic_poi B-timeRange I-timeRange I-timeRange I-timeRange I-timeRange I-timeRange I-timeRange
O O O O B-condition_description O B-city I-city B-state
O O B-city B-state
O O O O O B-condition_description O B-timeRange I-timeRange O B-city I-city B-country
